src/sdk-reference/bulk/import/import.js.md

```diff
@@ -4,7 +4,8 @@
 import(bulkData, options = {})  ⇒ object
 ```
 
-The bulk import allows to save a list of documents into a specific collection (belonging to a specific index). If a subset of the documents fails to save, a PartialError is triggered. The `data` argument passed to the method must specify the set of documents to import and must satisfy the [Elasticsearch Bulk API](https://www.elastic.co/guide/en/elasticsearch/reference/5.4/docs-bulk.html).
+The bulk import allows to save a list of documents into a specific collection (belonging to a specific index). If a subset of the documents fails to save, a <a href="https://docs.kuzzle.io/api-documentation/errors#partialerror">PartialError</a> is triggered. The `data` argument passed to the method must specify the set of documents to import and must satisfy the [Elasticsearch Bulk API]
```

Comment:
why a direct `a href`, and not a dynamic self-referencing link?

Reply:
Because the page does not yet exist in the new documentation
and I think is better to keep the reference of this link to the old documentation